src/core/ElementRegistry.js

```diff
@@ -0,0 +1,18 @@
+import BaseElementRegistry from 'diagram-js/lib/core/ElementRegistry';
+
+
+export class ElementRegistry extends BaseElementRegistry {
+
+    constructor (props) {
+        super(props);
+    }
+
+    /**
+     * TODO: Remove hotfix for handles
+     * @return {*}[]
+     */
+    getAll () {
+        return super.getAll().filter((element) => element.type !== 'handle');
```

Comment:
Good hotfix :grin: Improved behavior of handles (#99) should best be implemented soon.

Reply:
Thanks :D We interchanged that this will be your task 🙃. I've assigned you in the issue #99.

Reply:
:+1: